### DISCUSSION TITLE
.NET Runtime 6.0 support

### DISCUSSION THREAD

**Discussion by gordongreeff** [sonik-br/GDMENUCardManager#26] [Ideas]
Appreciate you have little time update, but any chance of supporting .NET Runtime 6.0 (in respect to 5.0 being end-of-life) ?

Thanks.

**Comment by sonik-br** [comment]
Hi. I plan on doing this but might take some time.
Still need to check if all the required libs works with net 6 and also check OS compatibility.

**Comment by sonik-br** [comment]
I've updated it to dot net 6.
There's a new version for win/osx/linux at the pre-releases.
It need to be tested under osx and linux before I commit it to stable.
